tests/unit/dependency-injection/schema-templates-loader-test.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Dependency_Injection;
+
+use Yoast\WP\SEO\Dependency_Injection\Schema_Templates_Loader;
+use Yoast\WP\SEO\Tests\Unit\TestCase;
+
+/**
+ * Class Schema_Templates_Loader_Test.
+ *
+ * @group dependency-injection
+ *
+ * @coversDefaultClass \Yoast\WP\SEO\Dependency_Injection\Schema_Templates_Loader
+ */
+class Schema_Templates_Loader_Test extends TestCase {
+
+	/**
+	 * Represents the instance to test.
+	 *
+	 * @var Schema_Templates_Loader
+	 */
+	protected $instance;
+
+	/**
+	 * Sets the instances to test.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->instance = new Schema_Templates_Loader();
+	}
+
+	/**
+	 * Tests the retrieval of the templates.
+	 *
+	 * @covers ::get_templates
+	 */
+	public function test_get_templates() {
+		$schema_directory = dirname( WPSEO_FILE ) . '/config/dependency-injection/../../src/schema-templates/';
```

Comment:
- it's the directory where the _templates_ live 
```suggestion
		$schema_template_directory = dirname( WPSEO_FILE ) . '/src/schema-templates/';
```

Reply:
also - why are you navigating to a config/dependency-injection folder only to navigate back and back again?

Reply:
This is because the `glob` in the `Schema_Template_Pass` gives exactly this directory. We can't use the constant there like we do here, because the path will be compiled during `composer install`. I would love to give the path you give, but that is way too difficult in this case.